components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -1,24 +1,39 @@
 import React from "react";
 import Image, { StaticImageData } from "next/image";
 
-import JavascriptIcon from "img/icons/👾.svg";
-import ReactIcon from "/img/icons/⚛️.svg";
-import PythonIcon from "/img/icons/🐍.svg";
-import AIIcon from "/img/icons/🤖.svg";
-import MLIcon from "/img/icons/🧠.svg";
-import RustIcon from "/img/icons/🦀.svg";
-import PhpIcon from "/img/icons/📄.svg";
-import CsharpIcon from "/img/icons/🎹.svg";
-import CIcon from "/img/icons/🥸.svg";
-import CppIcon from "/img/icons/🤓.svg";
-import TypeScriptIcon from "/img/icons/🔥.svg";
-import RubyIcon from "/img/icons/🌹.svg";
-import JavaIcon from "/img/icons/☕️.svg";
+import JavascriptIcon from "img/icons/javascript.svg";
+import ReactIcon from "/img/icons/react.svg";
+import PythonIcon from "/img/icons/python.svg";
+import AIIcon from "/img/icons/ai.svg";
+import MLIcon from "/img/icons/machine-learning.svg";
+import RustIcon from "/img/icons/rust.svg";
+import PhpIcon from "/img/icons/php.svg";
+import CsharpIcon from "/img/icons/c-sharp.svg";
+import CIcon from "/img/icons/c.svg";
+import CppIcon from "/img/icons/c-plus.svg";
+import TypeScriptIcon from "/img/icons/typescript.svg";
+import RubyIcon from "/img/icons/ruby.svg";
+import JavaIcon from "/img/icons/java.svg";
+import GolangIcon from "/img/icons/golang.svg";
 import topicNameFormatting from "lib/utils/topic-name-formatting";
 
-
 interface LanguagePillProps {
-  topic: "react" | "javascript" | "python" | "ML" | "AI" | "rust" | "ruby" | "c" | "cpp" | "csharp" | "php" | "java" | "typescript" | string;
+  topic:
+    | "react"
+    | "javascript"
+    | "python"
+    | "ML"
+    | "AI"
+    | "rust"
+    | "ruby"
+    | "c"
+    | "cpp"
+    | "csharp"
+    | "php"
+    | "java"
+    | "typescript"
+    | "golang"
+    | string;
```

Comment:
I have a review here similar to my review in #966 . Here it is creating a new type for all the available topic names instead of using the already existing `interests` array in `getInterestOptions`. We should have only 1 central source of truth and sync every usage with it.

Discussion link for full context:
https://github.com/open-sauced/insights/pull/966#discussion_r1133130848

It was already fixed there, so I don't know how this would go for this current PR. (Like if you would need to fix it as well or if you'd need to wait for the other PR to merge and continue on it or what)

Reply:
I merged the previous PR, so this one needs to be resolved to work with it

Reply:
Ok